examples/erc20/src/transfer.ts

```diff
@@ -0,0 +1,72 @@
+import {
+  Domain,
+  EVMAssetTransfer,
+  Environment,
+  Fungible,
+  Resource,
+  Transfer,
+} from "@buildwithsygma/sygma-sdk-core";
+import { Wallet, providers } from "ethers";
+
+export const GOERLI_CHAIN_ID = 5;
+export const SEPOLIA_CHAIN_ID = 11155111;
+export const ERC20_TOKEN_SYMBOL = "ERC20LRTest";
+
+export async function erc20Transfer(): Promise<void> {
+  const provider = new providers.JsonRpcProvider(
+    "https://rpc.goerli.eth.gateway.fm/"
+  );
+  const wallet = new Wallet(
+    "9574830b950b9ce5b66836e77ea18739258682ca08d4e65ca26b03dfe3742cf9",
+    provider
+  );
+  const assetTransfer = new EVMAssetTransfer();
+  await assetTransfer.init(provider, Environment.TESTNET);
+
+  const domains: Array<Domain> = assetTransfer.config.getDomains();
```

Comment:
No need to specify the return type here, as TS should be able to determine it from the types specified on `getDomains()`
```suggestion
  const domains = assetTransfer.config.getDomains();
```